data/json/monsters.json

```diff
@@ -2510,6 +2549,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_predator",
+    "name": "feral predator",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"brown_green",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":6,
+    "aggression":100,
+    "morale":100,
+    "speed":170,
```

Comment:
With speed 170, leap per 5 turns and vision 45/15, it's impossible to escape and hard to shoot it down while it comes running. Also, the 170 speed means it can easily leap and impale immediately after.